Moo::Role inflation to Moose doesn't create a type constraint, so 'isa'
fails. Email::Sender::Transport has been a role since before we started
using it, so just use an explicit 'does' instead of 'isa'.
---
 lib/Catalyst/View/Email.pm |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Catalyst/View/Email.pm b/lib/Catalyst/View/Email.pm
index d7e7c40..8c6055d 100644
--- a/lib/Catalyst/View/Email.pm
+++ b/lib/Catalyst/View/Email.pm
@@ -20,7 +20,7 @@ has 'mailer' => (
 
 has '_mailer_obj' => (
     is      => 'rw',
-    isa     => 'Email::Sender::Transport',
+    does    => 'Email::Sender::Transport',
     lazy    => 1,
     builder => '_build_mailer_obj',
 );
-- 
1.7.10.4

